contracts/common/Singleton.sol

```diff
@@ -1,11 +1,13 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity >=0.7.0 <0.9.0;
 
-/// @title Singleton - Base for singleton contracts (should always be first super contract)
-///         This contract is tightly coupled to our proxy contract (see `proxies/SafeProxy.sol`)
-/// @author Richard Meissner - <richard@gnosis.io>
+/**
+ * @title Singleton - Base for singleton contracts (should always be first super contract)
+ *        This contract is tightly coupled to our proxy contract (see `proxies/SafeProxy.sol`)
+ * @author Richard Meissner - @rmeissner
+ */
 contract Singleton {
-    // singleton always needs to be first declared variable, to ensure that it is at the same location as in the Proxy contract.
-    // It should also always be ensured that the address is stored alone (uses a full word)
+    // The singleton always has to be the first declared variable to ensure the same location as in the Proxy contract.
+    // Should also always be ensured the address is stored alone (uses a full word)
```

Comment:
`It should also always be ensured that the address is stored alone (uses a full word)` sounded better in my opinion, I don't understand this change

Reply:
I tried to keep the docs short and IMO words "it" and "that" are unnecessary and the context still can be understood

Reply:
IMO we shouldn't modify text that's already clear and well written, but I don't have a strong opinion here

Reply:
I would also keep the "It" here.